api/src/services/fields.ts

```diff
@@ -539,7 +541,7 @@ export class FieldsService {
 				}
 			}
 
-			if (hookAdjustedField.meta) {
+			if (hookAdjustedField.meta && !isSystemField(collection, hookAdjustedField.field)) {
```

Comment:
Should we add a comment here explaining why we are checking system fields here? May not be immediately obvious